tests/plugins/editorplaceholder/editorplaceholder.js

```diff
@@ -205,5 +219,101 @@
 		} );
 	};
 
+	// (#4253)
+	tests[ 'test placeholder loads correctly in full-page editor with bbcode plugin' ] = function() {
+		consoleErrorStub = sinon.stub( console, 'error' );
+
+		var runtimeErrorListener = function() {
+			resume( function() {
+				assert.fail( 'There should be no RunTime errors!' );
+			} );
+		};
+
+		window.addEventListener( 'error', runtimeErrorListener );
+
+		bender.editorBot.create( {
+			name: 'bbcodeFailTest',
+			config: {
+				extraPlugins: 'bbcode',
+				editorplaceholder: 'any',
+				fullPage: true,
+				on: {
+					instanceReady: function() {
+						resume( function() {
+							assert.isFalse( consoleErrorStub.called, 'There should be no errors during initialization' );
+						} );
+					}
+				}
+			}
+		}, function( bot ) {
+
+		} );
+
+		wait();
+	};
+
+	// (#4253)
+	tests[ 'test placeholder loads correctly in full-page editor with bbcode plugin and initial content' ] = function() {
+		bender.editorBot.create( {
+			name: 'bbcodeFailTestContent',
+			config: {
+				extraPlugins: 'bbcode',
+				editorplaceholder: 'any',
+				fullPage: true,
+				startupData: '<p>Initialized content</p>'
+			}
+		}, function( bot ) {
+			var editor = bot.editor;
+
+			assert.isFalse( editor.editable().hasAttribute( 'data-cke-editorplaceholder' ) );
+		} );
+	};
+
+	// (#4253)
+	tests[ 'test placeholder is visible with data filter to no content' ] = function() {
+		bender.editorBot.create( {
+			name: 'placeholderFilterToNoContent',
+			config: {
+				editorplaceholder: 'any',
+				fullPage: true,
+				startupData: '<p>Initialized content</p>'
+			}
+		}, function( bot ) {
+			var editor = bot.editor;
+
+			editor.on( 'toDataFormat', function ( evt ) {
+				evt.data.dataValue = '';
+			}, null, null, 16 );
+
+			editor.fire( 'change' );
```

Comment:
Why `change` event needs to be fired here (and in the next test too)?

Reply:
`editor.fire( 'change' );` is invoked just to be sure, that the newly added `toDataFormat` filter will be applied before assertion. No matter what `startupData` is.

In this case there is no `startupData` anyway, but I want to be sure that case from [#4253 (
 comment)](https://github.com/ckeditor/ckeditor4/issues/4253#issuecomment-689774158) is covered.

Reply:
Ok, I see that without `change` fire this test does not fail without a fix, so it makes sense as you mentioned :+1: